examples/30_extended/tasks_tutorial.py

```diff
@@ -106,8 +103,7 @@
 # Finally, it is also possible to list all tasks on OpenML with:
 
 ############################################################################
-tasks = openml.tasks.list_tasks()
-tasks = pd.DataFrame.from_dict(tasks, orient='index')
+tasks = openml.tasks.list_tasks(output_format='dataframe')
```

Comment:
didn't know this new feature. Are examples somehow tested?

Reply:
Yeah, that's pretty handy. Once we have closed all more important issues we should also make this the default behavior ;)

Examples are tested by running while building the documentation. They should make the unit test fail if they fail themselves.